### PR TITLE
GCB-252: Fix bubbles pulsing when close to edge

### PR DESCRIPTION
**Pull request by michallaskowski**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [x] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [x] The project does not generate any warnings or errors
- [x] The tests are passing
- [x] Branch name is in line with GitFlow ([WTF?](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/))

### CODE REVIEW

File: GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift [hunk 1]
```diff
@@ -44,15 +44,25 @@ class BonusScene: SKScene {
         }
     }
 
+    func areNodesToSmallWithAccumulatedFrame(accumulatedFrame: CGRect) -> Bool {
```
Comment (ochococo) [inline GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift]: Add tests?

**Comment by ochococo** [comment]
No more comments.

File: GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift (hunk 1 above)
Comment (ochococo) [inline GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift]: A typo `areNodesTooSmallWithAccumulatedFrame` : `To` -> `Too`

File: GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift [hunk 2]
```diff
@@ -14,6 +14,7 @@ class BonusScene: SKScene {
     private var viewModel: BonusWidgetViewModel!
     private let world = SKNode()
     var bubbleResizeDuration: NSTimeInterval = 15
+    var relativeScenePadding: (x: CGFloat, y: CGFloat) = (x: 0.05, y: 0.05)
```
Comment (ochococo) [inline GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift]: `CGPoint()` perhaps?

File: GrandCentralBoardTests/Widgets/Bonus/BonusSceneTests.swift [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+//  Created by Michał Laskowski on 29.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import XCTest
+import Nimble
+@testable import GrandCentralBoard
+
+
+final class BonusSceneTests: XCTestCase {
```
Comment (ochococo) [inline GrandCentralBoardTests/Widgets/Bonus/BonusSceneTests.swift]: 👏

**Comment by ochococo** [comment]
@michallaskowski Found some minor things. 🎾 Good job overall.

File: GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift (hunk 2 above)
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift]: I am not sure. It is not a size to use `CGSize`, not a vector for `CGVector`, and it is not a point. That's why I went with a tuple. I would prefer to stay with it, I think making it `CGPoint` would confuse someone reading the code.

**Comment by michallaskowski** [comment]
@ochococo Thanks :)

**Pull request merged**